codec/seata-codec-kryo/src/main/java/io/seata/codec/kryo/KryoCodec.java

```diff
@@ -27,7 +27,7 @@
 
     @Override
     public <T> byte[] encode(T t) {
-        if (t == null || !(t instanceof AbstractMessage)) {
+        if (!(t instanceof AbstractMessage)) {
             throw new IllegalArgumentException("message is null");
```

Comment:
The exception message should be changed as well.how about "message is illegal".

Reply:
Yes, I didn't notice the message.